pkg/reconciler/v1alpha1/revision/config/observability.go

```diff
@@ -71,5 +74,10 @@ func NewObservabilityFromConfigMap(configMap *corev1.ConfigMap) (*Observability,
 	} else {
 		oc.LoggingURLTemplate = defaultLogURLTemplate
 	}
+
+	if rlt, ok := configMap.Data["logging.request-log-template"]; ok {
+		oc.RequestLogTemplate = rlt
```

Comment:
This should parse the Go template and return an error if it doesn't work.

Reply:
Part of this, btw is to enable us to have unit tests that validate the example we have checked in is a valid template.  As is, we'll check that it's a string :(

Reply:
Good point - but even a valid go template doesn't really mean that the template will execute correctly at runtime. So, I was thinking of having these as soft failures and anything invalid would suppress request logs.

That being said, I will change this part to parse the template.

Reply:
Actually, I take back what I am saying :) I cannot validate the template until I pass all the extra functions I pass to it. Example:

```
funcMap := template.FuncMap{"env": utils.GetCachedEnv}
template, err := template.New("requestLog").Funcs(funcMap).Parse(templateStr)
```

I can move this whole thing into the config, but then config would be responsible for more than what it should be doing. I will keep this as is. Let me know if you have concerns there.